enterprise/backend/src/metabase_enterprise/content_translation/dictionary.clj

```diff
@@ -55,26 +65,27 @@
        (adjust-index index)))
 
 (defn- process-rows
-  "Format, trim, validate rows. Takes the vectors from a csv and returns a map with the shape
+  "Format and validate rows. Takes the vectors from a csv and returns a map with the shape
   {:translations [{:locale :msgid :msgstr}]
    :errors       [string]}.
   The :seen set is returned but not meant for consumption."
   [rows]
-  (reduce (fn [state [index row]]
-            (let [[locale msgid msgstr & extra] row
-                  translation                   {:locale locale
-                                                 :msgid  (str/trim msgid)
-                                                 :msgstr (str/trim msgstr)}
-                  errors                        (cond-> (row-errors state index translation)
-                                                  (seq extra) (conj (wrong-row-shape index)))]
-              (cond-> (-> state
-                          (update :seen conj (dissoc translation :msgstr))
-                          (update :translations conj translation))
-                (seq errors) (update :errors into errors))))
-          {:seen         #{}
-           :errors       []
-           :translations []}
-          (map-indexed vector rows)))
+  (let [formatted-rows (map format-row rows)]
+    (reduce (fn [state [index row]]
+              (let [[locale msgid msgstr & extra] row
+                    translation                   {:locale locale
+                                                   :msgid  msgid
+                                                   :msgstr msgstr}
+                    errors                        (cond-> (row-errors state index translation)
+                                                    (seq extra) (conj (wrong-row-shape index)))]
+                (cond-> (-> state
+                            (update :seen conj (dissoc translation :msgstr))
+                            (update :translations conj translation))
+                  (seq errors) (update :errors into errors))))
+            {:seen         #{}
+             :errors       []
+             :translations []}
+            (map-indexed vector formatted-rows))))
```

Comment:
I think the rows will already be vectors here. So you could just use `map-indexed` on line 73 or get rid of the let and do `(map-indexed format-row rows)` on this line